oathkeeper/12_ory_cloud_keto/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+services:
+  nginx:
+    image: nginx
+    ports:
+      - '8080:80'
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./nginx/index.html:/app/html:ro
+    networks:
+      - intranet
+
+  oathkeeper:
+    image: oryd/oathkeeper:v0.38
+    depends_on:
+      - kratos
```

Comment:
```suggestion
```

I think this was from the other example?

Reply:
Yep